packages/xstate-react/src/useService.ts

```diff
@@ -11,20 +11,20 @@ export function getServiceSnapshot<
     : service.machine.initialState;
 }
 
+/**
+ * @deprecated Use `useActor` instead.
+ *
+ * @param service The interpreted machine
+ * @returns A tuple of the current `state` of the service and the service's `send(event)` method
+ */
 export function useService<
   TContext,
   TEvent extends EventObject,
   TTypestate extends Typestate<TContext> = { value: any; context: TContext }
 >(
   service: Interpreter<TContext, any, TEvent, TTypestate>
 ): [State<TContext, TEvent, any, TTypestate>, PayloadSender<TEvent>] {
-  if (process.env.NODE_ENV !== 'production' && !('machine' in service)) {
-    throw new Error(
-      `Attempted to use an actor-like object instead of a service in the useService() hook. Please use the useActor() hook instead.`
-    );
-  }
```

Comment:
IMHO the tip was still useful here and we could keep it as long as `useService` exists.

Reply:
Done in ae2f97e5ba9423c354b626a7ea31eb7a4ab2c487